Src/Support/GoogleApis.Auth.PlatformServices_Shared/OAuth2/Pkcs8.cs

```diff
@@ -225,7 +222,9 @@ public static RSAParameters DecodeRsaParameters(string pkcs8Base64)
             const string PrivateKeySuffix = "-----END PRIVATE KEY-----";
 
             Utilities.ThrowIfNullOrEmpty(pkcs8Base64, "pkcs8PrivateKey");
-            var base64PrivateKey = pkcs8Base64.Replace(PrivateKeyPrefix, "").Replace(PrivateKeySuffix, "");
+            pkcs8Base64 = pkcs8Base64.Trim();
+            string base64PrivateKey = pkcs8Base64.StartsWith(PrivateKeyPrefix) && pkcs8Base64.EndsWith(PrivateKeySuffix) ?
```

Comment:
Do we ever expect to get it without the prefix/suffix? If not, I'd throw an exception if we see that. But I don't mind being liberal too much.